app/assets/javascripts/admin/api_flow_types.js

```diff
@@ -163,6 +163,14 @@ export type APITaskTypeType = {
   +settings: APISettingsType,
 };
 
+export type APICreateTaskType = {
```

Comment:
Let's try to avoid this type code duplication and wait for @fm3 answer :)

Reply:
@daniel-wer ok, that was just a quick solution i came up with @philippotto. But your suggestion makes way more sense  :)